README.md

```diff
@@ -1 +1,66 @@
-# backend-training
\ No newline at end of file
+## Details
+
+1. I have created a file of index.js where i have taken the input for operands from user and passing it to the functions which is created inside math.js which internally calls the lodash arithmetic function for the calculations,and then the output of the function is stored in the solution.csv file.
+
+## Installation:
+
+1. Clone the repository to your local machine:
+   git clone https://github.com/kushagra-successive/backend-training.git
+2. Install dependencies:
```

Comment:
You don't need to install every dependency one by one. You can just do npm i or npm install